libyul/AsmAnalysis.cpp

```diff
@@ -705,6 +706,16 @@ bool AsmAnalyzer::validateInstructions(evmasm::Instruction _instr, SourceLocatio
 		);
 	};
 
+	// The errors below are meant to be issued when processing an undeclared identifier matching a builtin name
+	// present on the default EVM version but not on the currently selected one,
+	// since the other `validateInstructions()` overload uses the default EVM version.
+	if (_instr == evmasm::Instruction::BLOBHASH)
+	{
+		// TODO: Upgrade this assertion to an error, similar to the ones above, when Cancun becomes the default EVM version.
```

Comment:
```suggestion
		// TODO: Upgrade this assertion to an error, similar to the ones below, when Cancun becomes the default EVM version.
```
Annoying nitpick, but I could not find anything else :D

Reply:
Ops! Thanks ;)